drivers/eeprom/include/eeprom_driver.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2018 Inria
+ *               2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    drivers_eeprom Common EEPROM interface
+ * @ingroup     drivers_storage
+ *
+ * This module provides a common API for accessing EEPROM (both internal and
+ * external), as well as common code and convenience interfaces.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Interface definition of EEPROM drivers
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @author      Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ */
+
+#ifndef EEPROM_DRIVER_H
+#define EEPROM_DRIVER_H
+
+#include "eeprom.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Driver interface for EEPROM devices
+ *
+ * This interface must be implemented by every EEPROM device driver. The
+ * function pointers in @ref eeprom_driver_t::set and
```

Comment:
> ```warning: unable to resolve reference to `eeprom_driver_t::set' for \ref command```

Reply:
Oh yes, I removed that function. Thanks for noticing!

Fixed :-)